tests/integration/test_backends.py

```diff
@@ -139,6 +172,27 @@ def test_runtime_backend_errors_handled(ws, query):
     assert result == "PASSED"
 
 
+def test_runtime_backend_handles_concurrent_append(ws, make_random, make_table) -> None:
```

Comment:
@nfx : I copied this from the tests above to integration test the runtime backend. However, it is not really the `RuntimeBackend`. Is this the correct approach for integration testing the runtime backend? I would have introduced a local Spark session using `pytest-spark`.